documentation/content/admin-states.html

```diff
@@ -4,197 +4,202 @@
 ---
 
 <p>
-  Clients route requests to distributors using the distribution algorithm.
-  Node states are input to the algorithm.
-  Distributors use the distribution algorithm to keep bucket copies on correct
-  storage nodes. To do this, it needs to know which storage nodes are available
-  to hold copies and what their capacities are.
-  Service layer nodes may split requests between different partitions, in which
-  case it needs to know what partitions are available.
-  To achieve this, state is tracked for all the major components of the content layer:
+Clients route requests to distributors using the distribution algorithm.
+Node states are input to the algorithm.
+Distributors use the distribution algorithm to keep bucket copies on correct storage nodes.
```

Comment:
As a follow-up for later, for historical reasons we are currently using (at least) 4 terms for content nodes somewhat interchangeably in our documentation:
1) storage nodes
2) service layer nodes
3) content nodes
4) search nodes

We should find one and stick to it

Reply:
very much agree, sir. imho, _content_ node is what is configured in services.xml, and we can use proton in _search_ context. So for me, number 3 is the winner, but open for discusssion